### PR TITLE
Remove references to papers

### PR DESCRIPTION
**Pull request by tkoeppe**

### CODE REVIEW

**Comment by tkoeppe** [comment]
@jwakely: How about this?

**Review by jwakely — APPROVED** [review]

File: fundamentals-ts.html [hunk 1]
```diff
@@ -1652,80 +1651,70 @@ <h1>Contents</h1>
...
         <td>Invocation type traits</td>
         <td><cxx-ref to="meta.trans.other"><a title="meta.trans.other" href="#meta.trans.other">6.3.2</a></cxx-ref></td>
         <td><code>invocation_type</code></td>
         <td>201406</td>
         <td><code>&lt;experimental/type_traits&gt;</code></td>
       </tr>
       <tr>
-        <td>N4502</td>
-        <td>The C++ Detection Idiom</td>
+        <td>Detection metaprograms</td>
```
Comment (jensmaurer) [inline fundamentals-ts.html:1675]: I think this is not the right summary of the feature. Isn't this `void_t` from Walter?

Comment (tkoeppe) [reply fundamentals-ts.html:1675]: It's not only `void_t`, but `is_detected` and `nonesuch` etc.: https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2015/n4502.pdf

Any suggestions for how to call this?

Comment (tkoeppe) [reply fundamentals-ts.html:1675]: See https://cplusplus.github.io/fundamentals-ts/v3.html#meta.detect

Comment (jensmaurer) [reply fundamentals-ts.html:1675]: Ah, ok. Then, the summary does look plausible. (Not that anyone would deeply care.)

Comment (tkoeppe) [reply fundamentals-ts.html:1675]: True, but I did just make those names up on the fly, so the review is very much welcome.

**Pull request merged**